benchmarking/commons/baselines.py

```diff
@@ -117,6 +117,14 @@ def _is_fcnet(config_space: Dict[str, Any]) -> bool:
     return fcnet_keys.issubset(set(config_space.keys()))
 
 
+def _to_be_converted(domain: Domain) -> bool:
```

Comment:
All of this is to replace a `choice` definition by an `ordinal` for the fcnet configuration space. The original benchmark has a 6-valued `choice`, but this is not good for BO.